### PR TITLE
First step in localization

### PR DESCRIPTION
**Pull request by darkwing**
Implements `gettext` wrappers around l10n content.

### CODE REVIEW

File: wp-web-push-admin.php [hunk 1]
```diff
@@ -26,34 +26,34 @@ public function options() {
       } else if ($_POST['webpush_title'] === 'custom') {
         update_option('webpush_title', $_POST['webpush_title_custom']);
       } else {
-        wp_die('Wrong value for the Notification Title');
+        wp_die(__('Wrong value for the Notification Title'));
       }
 ?>
-<div class="updated"><p><strong>Settings saved.</strong></p></div>
+<div class="updated"><p><strong><?php echo __('Settings saved'); ?></strong></p></div>
```
Comment (marco-c) [inline wp-web-push-admin.php]: Nit: you can use `_e` directly instead of echoing `__(`

Comment (darkwing) [inline wp-web-push-admin.php]: I know, I thought I'd try to be consistent;  I can change to `_e` though  _shrugs_

File: wp-web-push-admin.php [hunk 2]
```diff
@@ -26,34 +26,34 @@ public function options() {
...
-<th scope="row">Notification Title</th>
+<th scope="row"><?php echo __('Notification Title'); ?></th>
 <td>
 <fieldset>
-<label><input type="radio" name="webpush_title" value="blog_title" <?php echo $title_option === 'blog_title' ? 'checked' : '' ?> /> Use the Site Title: <b><?php echo get_bloginfo('name'); ?></b></label><br />
-<label><input type="radio" name="webpush_title" value="custom" <?php echo $title_option !== 'blog_title' ? 'checked' : '' ?> /> Custom:</label>
-<input type="text" name="webpush_title_custom" value="<?php echo $title_option !== 'blog_title' ? $title_option : 'Write here the title' ?>" class="long-text" />
+<label><input type="radio" name="webpush_title" value="blog_title" <?php echo $title_option === 'blog_title' ? 'checked' : '' ?> /> <?php echo __('Use the Site Title'); ?>: <b><?php echo get_bloginfo('name'); ?></b></label><br />
+<label><input type="radio" name="webpush_title" value="custom" <?php echo $title_option !== 'blog_title' ? 'checked' : '' ?> /> <?php echo __('Custom'); ?>:</label>
+<input type="text" name="webpush_title_custom" value="<?php echo $title_option !== 'blog_title' ? $title_option : __('Your custom title') ?>" class="long-text" />
```
Comment (marco-c) [inline wp-web-push-admin.php]: `esc_attr_e` should be used here

File: wp-web-push-admin.php (hunk 1 above)
Comment (marco-c) [inline wp-web-push-admin.php]: It's a "nit", so you can safely ignore it.

**Comment by darkwing** [comment]
This is ready to merge; we can figure out the po/mo generation later

File: wp-web-push/wp-web-push-admin.php [hunk 3]
```diff
@@ -26,34 +28,34 @@ public function options() {
...
+<label><input type="radio" name="webpush_title" value="custom" <?php echo $title_option !== 'blog_title' ? 'checked' : '' ?> /> <?php _e('Custom', 'wpwebpush'); ?>:</label>
+<input type="text" name="webpush_title_custom" value="<?php echo $title_option !== 'blog_title' ? $title_option : __('Your custom title', 'wpwebpush') ?>" class="long-text" />
 </fieldset>
 </td>
 </tr>
 </table>
 
 <p class="submit">
-<input type="submit" name="submit" id="submit" class="button button-primary" value="Save Changes" />
+<input type="submit" name="submit" id="submit" class="button button-primary" value="<?php _e('Save Changes', 'wpwebpush'); ?>" />
```
Comment (marco-c) [inline wp-web-push/wp-web-push-admin.php:58]: I think we should use `esc_attr_e` for attributes, no?

File: wp-web-push/wp-web-push-admin.php [hunk 4]
```diff
@@ -26,34 +28,34 @@ public function options() {
...
 <tr>
-<th scope="row">Notification Title</th>
+<th scope="row"><?php _e('Notification Title', 'wpwebpush'); ?></th>
 <td>
 <fieldset>
-<label><input type="radio" name="webpush_title" value="blog_title" <?php echo $title_option === 'blog_title' ? 'checked' : '' ?> /> Use the Site Title: <b><?php echo get_bloginfo('name'); ?></b></label><br />
-<label><input type="radio" name="webpush_title" value="custom" <?php echo $title_option !== 'blog_title' ? 'checked' : '' ?> /> Custom:</label>
-<input type="text" name="webpush_title_custom" value="<?php echo $title_option !== 'blog_title' ? $title_option : 'Write here the title' ?>" class="long-text" />
+<label><input type="radio" name="webpush_title" value="blog_title" <?php echo $title_option === 'blog_title' ? 'checked' : '' ?> /> <?php _e('Use the Site Title', 'wpwebpush'); ?>: <b><?php echo get_bloginfo('name'); ?></b></label><br />
+<label><input type="radio" name="webpush_title" value="custom" <?php echo $title_option !== 'blog_title' ? 'checked' : '' ?> /> <?php _e('Custom', 'wpwebpush'); ?>:</label>
```
Comment (marco-c) [inline wp-web-push/wp-web-push-admin.php:50]: I'm not sure how this works, but probably some of these strings will already be localized in WordPress (for example "Custom", "Save Changes", "Saved settings", etc.).
We can figure out how to deal with those later (we can re-localize them all, or we can use the WordPress localized strings).

**Pull request merged**